docs/SqueezingPerformance.md

```diff
@@ -0,0 +1,55 @@
+# Squeezing Performance
+
+Performance should already be good enough for 99% of people, but to squeeze out more when necessary, there are some
+tricks.
+
+These are ordered from least to most inconvenient (and drastic)
+
+## `@IgnoreError`
+
+A huge performance improvement can be gained by telling JNR-FFI to note save the last errno. By default, JNR-FFI will
```

Comment:
note should be not

Reply:
I already merged but will fix that typo in place.